src/orchard/stacktrace/parser/aviso.clj

```diff
@@ -0,0 +1,105 @@
+(ns orchard.stacktrace.parser.aviso
+  "Parser for stacktraces in the Aviso format."
+  {:added "0.10.1"}
+  (:require [clojure.java.io :as io]
+            [instaparse.core  :as insta :refer [defparser]]
+            [orchard.misc :refer [safe-read-edn]]
+            [orchard.stacktrace.parser.util :as util]))
+
+(def ^{:added "0.10.1" :private true}
```

Comment:
The `:added` metadata is not needed for private APIs.

Reply:
Fixed